tools/data_converter/s3dis_data_utils.py

```diff
@@ -99,15 +107,51 @@ def process_single_scene(sample_idx):
                 'instance_mask', f'{self.split}_{sample_idx}.bin')
             info['pts_semantic_mask_path'] = osp.join(
                 'semantic_mask', f'{self.split}_{sample_idx}.bin')
+            info['annos'] = self.get_bboxes(points, pts_instance_mask,
+                                            pts_semantic_mask)
 
             return info
 
         sample_id_list = sample_id_list if sample_id_list is not None \
             else self.sample_id_list
-        with futures.ThreadPoolExecutor(num_workers) as executor:
+        with futures.ThreadPoolExecutor(1) as executor:
```

Comment:
Please use `num_workers` as thread number (or if there is any reason we can't use multi-threading here?)